enhancements/installer/install_ca_management.md

```diff
@@ -0,0 +1,164 @@
+components to know whether they should be consuming this value or not.
+
+The most direct consequences of this are that today if a customer does a disconnected install, 
+their imagestream imports from the mirror registry will fail.  In particular this means the 
+oauth-proxy imagestream, which is consumed by many OLM operators, will fail and thus those 
+operators will fail to function, until/unless the admin explicitly configures the CA in the image 
+config resource and then re-imports the imagestream.
+
+While this could theoretically be solved by the imagestream component trusting the additional CA (
+for all hosts?) explicitly, it would make more sense if we used the information the admin is 
+providing us at install time to correctly configure the cluster as if they had done it themselves, 
+rather than introducing additional sources of (potentially conflicting) configuration 
+information.
+
+## Motivation
+
+See summary above, motivation is to make the install experience clearer with less potential for misconfiguration and conflicting settings.
+
+### Goals
+
+- Post install the cluster can not only pull images from an alternate registry, but imagestreams can import from it, builds can pull from it, and the internal registry can perform pullthrough against it
+- Post install the registry CAs provided by the admin are populated into the configuration api that is intended for this purpose
+
+### Non-Goals
+
+- Refactoring the additionalTrustBundle field into many unique fields for different CA usages
+- Addressing the current proxy CA behavior in which the additionalTrustBundle is sometimes populated into a proxy configuration and other times not (depending on whether a proxy host is supplied)
+- Adding generic cluster configuration fields to the installer that are not required to bootstrap the cluster and ensure consistent use of cluster config apis
+
+
+## Proposal
+
+The installer should populate the 
+[image config resource](https://github.com/openshift/api/blob/master/config/v1/types_image.go) in the same way the 
+[product documentation](https://docs.openshift.com/container-platform/4.5/openshift_images/image-configuration.html#images-configuration-cas_image-configuration) 
+recommends.  The installer 
+can use the value of the additionalTrustBundle field to populate this api.  Similar to how the 
+value is used for the proxy config when a proxy host is supplied, the value can be used to 
+populate the image config resource when a mirror registry is in use.
+
+In addition, the downstream components that potentially communicate with the mirror registry should add logic which confirms they can successfully access the mirror registry.  Specifically, for any registry defined in the image content source policy, the components should be able to access the registry using the supplied CAs.  If they cannot, they should mark themselves degraded.
+
```

Comment:
Secondly, currently the MCO sets the node's trust to the CA in additional trust bundle and not just the trust for CRI-O to pull certain images from registries.

So are we recommending that the MCO change that behavior too reduce the trust from node for all to just for CRI-O? Would that be backward compatible?

Reply:
No.  I assumed it was only being provided to the nodes for CRIO(in which case the MCO behavior can be replaced by the image config resource CA distribution behavior), but if it's being put in the generic system trust for the node then as you say, we probably can't change that behavior, so we'll have to live with two mechanisms distributing trust to the nodes (one to the system trust for the node, and the other to the CRIO registry trust in `/etc/docker/certs.d`).